app/bundles/ReportBundle/Form/Type/DynamicFiltersType.php

```diff
@@ -57,11 +57,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             $args['data'] = ((int) $options['data'][$definition['alias']] == 1);
                         }
                         break;
+                    case 'date':
+                        $type           = 'date';
+                        $args['input']  = 'string';
+                        $args['widget'] = 'single_text';
+                        $args['format'] = 'y-M-d';
+                        $args['attr']['class'] .= ' datepicker';
+                        break;
                     case 'datetime':
                         $type           = 'datetime';
                         $args['input']  = 'string';
                         $args['widget'] = 'single_text';
-                        $args['format'] = 'Y-m-d H:i:s';
+                        $args['format'] = 'y-M-d H:i:s';
```

Comment:
Are you sure about this?

`y | A two digit representation of a year`
`M | A short textual representation of a month, three letters`

So an example you want is `17-Jan-25 01:01:01`?